conference/views/__init__.py

```diff
@@ -69,48 +68,8 @@ def speaker_xml(request, slug, speaker, full_access, talks):
     }
 
 
-@render_to_template('conference/talk.html')
-@talk_access
-def talk(request, slug, talk, full_access, talk_form=None):
-    conf = models.Conference.objects.current()
-    if talk_form is None:
-        talk_form = utils.dotted_import(settings.FORMS['AdditionalPaperSubmission'])
-    if request.method == 'POST':
-        if not full_access:
-            return http.HttpResponseBadRequest()
-        if conf.cfp():
-            data = request.POST
-        else:
-            data = request.POST.copy()
-            data['level'] = talk.level
-            data['duration'] = talk.duration
-            data['language'] = talk.language
-            data['type'] = talk.type
-            data['tags'] = ','.join([ x.name for x in talk.tags.all() ])
-        form = talk_form(data=data, files=request.FILES, instance=talk)
-        if not conf.cfp() and not data['tags'] and 'tags' in form.fields:
-            # The CFP and 'closed and we are editing a talk without tags,
-            # it is not' normally possible since the tags are required;
-            # we're probably editing a talk inserted through admin, and if that
-            # 's the case does not make sense to derail the form validation.
-            form.fields['tags'].required = False
-        if form.is_valid():
-            talk = form.save()
-            messages.info(request, 'Your talk has been modified.')
-            return HttpResponseRedirectSeeOther(reverse('conference-talk', kwargs={'slug': talk.slug}))
-    else:
-        form = talk_form(instance=talk)
-
-    return {
-        'form': form,
-        'full_access': full_access,
-        'social_image_url': request.build_absolute_uri(
```

Comment:
I think this needs to be added to the new talk view...

Reply:
The social card that Patrick made? I thought it was added in an earlier change - if not, I'll add it here as part of the changes.